modules/custom/az_migration/az_migration.info.yml

```diff
@@ -9,3 +9,4 @@ dependencies:
  - drupal:migrate_tools
  - drupal:migrate_drupal
  - drupal:media_migration
+ - az_carousel
```

Comment:
Maybe we should omit this (for consistency) and just declare our destination module dependencies in the migrations themselves.
```suggestion
```